DigitalLearningSolutions.Data.Tests/Services/RegistrationServiceTests.cs

```diff
@@ -41,15 +60,59 @@ public void Setup()
             );
             A.CallTo(() => registrationDataService.RegisterDelegate(failingRegistrationModel)).Returns("-1");
 
-            registrationService = new RegistrationService(registrationDataService, passwordDataService, emailService,
-                centresDataService);
+            registrationService = new RegistrationService
+            (
+                registrationDataService,
+                passwordDataService,
+                emailService,
+                centresDataService
+            );
+        }
+
+        [Test]
+        public void Registering_delegate_with_approved_IP_registers_delegate_as_approved()
+        {
+            // When
+            registrationService.RegisterDelegate(testRegistrationModel, "localhost", "123.456.789.100");

```

Comment:
It might be nicer to write something like this so the person that reads the test doesn't need to scroll all the way up to see whether the IP actually has a correct prefix :)

```
// Given 
var clientIP = approvedIPPrefix + ".100";

// When
etc
```